### PR TITLE
ISSUE-23 #Fixed deprecated import

### PR DESCRIPTION
**Pull request by Shilpa-Perera**
SSlContextBuilder has been deprecated
**Solution** 
change ```org.apache.http.conn.ssl.SSLContexts``` to ```org.apache.http.ssl.SSLContexts``` in CustomHttpClient.java class

Additionally some typos in the comments of CustomHttpClient.java class was fixed.
@authorjapps could you review ?

### CODE REVIEW

**Review by authorjapps — APPROVED** [review]

**Comment by Shilpa-Perera** [comment]
@nirmalchandra could you please review ?

**Review by nirmalchandra — APPROVED** [review]

**Comment by nirmalchandra** [comment]
> @nirmalchandra could you please review ?

yep, thanks. 
Looks fine to me. You can merge it now.

Also can you rerun your scenario to check all working as exepcted?

**Comment by Shilpa-Perera** [comment]
> Also can you rerun your scenario to check all working as exepcted?

@nirmalchandra @authorjapps I reran all the tests and all working as expected. Since I do not have the write access to this repo I can not merge the PR. Could you give me access?

**Comment by nirmalchandra** [comment]
> > Also can you rerun your scenario to check all working as exepcted?
> 
> @nirmalchandra @authorjapps I reran all the tests and all working as expected. Since I do not have the write access to this repo I can not merge the PR. Could you give me access?

Merged now.

That's wierd, let me add you to the repo as collaborator now.

**Comment by nirmalchandra** [comment]
> > Also can you rerun your scenario to check all working as exepcted?
> 
> @nirmalchandra @authorjapps I reran all the tests and all working as expected. Since I do not have the write access to this repo I can not merge the PR. Could you give me access?

You can now join via this link:
https://github.com/authorjapps/zerocode-hello-world/invitations

**Pull request merged**
